chapter2/tests/t12/nginx.conf

```diff
@@ -0,0 +1,74 @@
+user nginx nginx;
+worker_processes 1;
+error_log /tmp/camp/error.log notice;
+error_log /tmp/camp/error_debug.log debug;  
+pid /run/nginx.pid;
+
+events {
+	worker_connections 1024;
+}
+
+http {
+
+	log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+					  '$status $body_bytes_sent "$http_referer" '
+					  '"$http_user_agent" "$http_x_forwarded_for"';
+
+	access_log  /tmp/camp/access.log main;
+	keepalive_timeout   65s;
+	types_hash_max_size 4096;
+	include             /etc/nginx/mime.types;
+	default_type        application/octet-stream;
+		index index.html;
```

Comment:
format file properly. if you keep columns starting in the same position - then do it.
use tabs vs spaces.

Reply:
> format file properly. if you keep columns starting in the same position - then do it. use tabs vs spaces.


@dmitry-mightydevops 
Done